### DISCUSSION TITLE
Flatpak

### DISCUSSION THREAD

**Discussion by germaineman** [spacedriveapp/spacedrive#2749] [General]
Spacedrive would be a dream come true for me. If only I could install it on my Fedora powered desktop computer. Either Flatpak or .rpm would be ideal

**Comment by techwithjake** [comment]
Completely agree.

In the mean time, I used `alien` to convert from DEB to RPM and it seems to be working OK thus far on Tumbleweed.

[Small tutorial on how to use alien](https://www.makeuseof.com/how-to-convert-between-linux-packages-using-alien/)
